packages/api/src/api/client/people.ts

```diff
@@ -0,0 +1,26 @@
+import { Result } from "@formbricks/types/errorHandlers";
+import { NetworkError } from "@formbricks/types/errors";
+import { makeRequest } from "../../utils/makeRequest";
+import { TPerson, TPersonUpdateInput } from "@formbricks/types/people";
+
+export class PeopleAPI {
+  private apiHost: string;
+  private environmentId: string;
+
+  constructor(apiHost: string, environmentId: string) {
+    this.apiHost = apiHost;
+    this.environmentId = environmentId;
+  }
+
```

Comment:
please also add a `create` method.
We need to check how we use this endpoints in the future. Might be that we also need a get endpoint?